devices/MulticastDNS/Package/EndianBitConverter.cs

```diff
@@ -0,0 +1,257 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Iot.Device.MulticastDNS.Package
```

Comment:
There indeed no interface and no way to automatically select one or the other. Indeed, networking is always big, so you should safely be able to use the ones from big conversion.
Regarding your question on adjusting the existing, library, short answer is no. Because i's a compatible lib from the .NET big brother: https://github.com/dotnet/runtime/tree/main/src/libraries/System.Private.CoreLib/src/System/Buffers/Binary
If you want to add something more dynamic to select big/little, you can do this in this project but please, don't change the existing binary primitives.

Reply:
Ok, won't touch the BinaryPrimitives repo :-)

For this library only big is needed but I already have another case in protobuf which also contains a similar class. There it is needed since you could send data to or receive data from an achitecture of opposite endianness.

So to me feels there is use for such an interface but question is then, where to put it. Putting it here doesn't make sense from protobuf perspective nor from the other way around...